atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/cache/ReadOnlyTransactionScopedCacheTest.java

```diff
@@ -33,12 +33,15 @@
 import java.util.Set;
 import java.util.function.Function;
 import java.util.stream.Collectors;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
I'd prefer if we adjusted our Mock usage rather than throwing a `Strictness.LENIENT` everywhere. Let me know if it's not feasible.

Reply:
I removed the annotation. The default is `strict_stub`. I assume that is what you want.